Database/Corrections/sodQuestFixes.lua

```diff
@@ -3168,6 +3168,30 @@ function SeasonOfDiscovery:LoadQuests()
             [questKeys.requiredSpell] = -410107,
             [questKeys.zoneOrSort] = sortKeys.SHAMAN,
         },
+        [90214] = {
+            [questKeys.name] = "Water Shield",
+            [questKeys.startedBy] = {{3272,3273,3394},{3642}},
```

Comment:
For the "Regeneration" and "Savage Roar" rune, we (aka me) did not add the NPCs to receive the key 🤔 But we (me again) did also not add a description how to receive the key in the objectivesText.

I say we remove the NPCs here as well, but keep description and also add such a description for the runes listed above.

Moreover `152608` and `152618` are also valid Kolkar Booties.

```suggestion
            [questKeys.startedBy] = {nil,{152608,152618,3642}},
```

Reply:
Fixed in: https://github.com/Questie/Questie/pull/5610/commits/9df6737c3ece0b93e05f8449b1800e640518c8ee